packages/types/lib/logs/messages.ts

```diff
@@ -31,7 +31,7 @@ export type MessageState = 'waiting' | 'running' | 'success' | 'failed' | 'timeo
  */
 export interface MessageOpSync {
     type: 'sync';
-    action: 'pause' | 'unpause' | 'run' | 'run_full' | 'cancel' | 'init';
```

Comment:
why renaming `run_full` but keep `run`

Reply:
`run` is the actual execution wether it's full or incremental